test/old/testdir/test_ins_complete.vim

```diff
@@ -2890,13 +2870,240 @@ func Test_complete_fuzzy_match()
   delfunc OnPumChange
   delfunc Omni_test
   delfunc Comp
-  delfunc CompAnother
   unlet g:item
   unlet g:word
-  unlet g:selected
   unlet g:abbr
 endfunc
 
+func Test_complete_fuzzy_collect()
+  new
+  redraw  " need this to prevent NULL dereference in Nvim
```

Comment:
`curwin->w_grid` is not allocated until a redraw, so without this `:redraw` there is a NULL dereference in `pum_display()`. Not sure if that should be fixed, as it doesn't really happen in practice.